src/memary/agent/base_agent.py

```diff
@@ -307,7 +367,7 @@ def get_response(self) -> str:
         Returns:
             str: response from the RAG model
         """
-        llm_message_chatgpt = self._change_llm_message_chatgpt()
+        llm_message_chatgpt = self._change_llm_message_chat()
```

Comment:
should we not retrieve result using RAG from llama3 as default?

Reply:
It follows the user selction of llm model, so whichever model the user chooses is the model used for all llm applications